lazy_theta_star_b/include/lazy_theta_star_b/lazy_theta_star_b.h

```diff
@@ -0,0 +1,176 @@
+/*********************************************************************
+ *
+ * Software License Agreement (BSD License)
```

Comment:
Can this be Apache 2.0? The vast majority of the Nav2 stack is Apache and we're trying to shift over to that for the long term where possible. Its not functionally any more restrictive but it does also protect your IP more.

Reply:
I don't really mind one way or another too much, just a note